frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1021,8 +1028,12 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 campaignCostsBreakdown,
                 marketingContext
             ): WebAnalyticsTile[] => {
-                const { createMarketingDataWarehouseNodes, marketingCompareFilter, marketingDateFilter } =
-                    marketingContext
+                const {
```

Comment:
We should refactor this soon to be a central dict with the possible tile definitions, and each page can build the order it wants.